tests/sgf/non_fox_game.sgf

```diff
@@ -0,0 +1,282 @@
+(;FF[4]
+CA[UTF-8]
+GM[1]
+DT[2023-02-22]
+PC[OGS: https://online-go.com/game/51301385]
+GN[Friendly Match]
+PB[aTypicalHuman]
+PW[JohnPS]
+BR[7k]
+WR[5k]
+TM[900]OT[3x30 byo-yomi]
+RE[B+3.5]
+SZ[19]
+C[Adjusted komi to be detected in testing]
+KM[11]
+RU[Japanese]
+;B[pq]
+(;W[dc]
```

Comment:
Out of curiosity, why is every single move a new branch?

Reply:
I wondered that too but I'm not sure. I found it inside one of the KaTrain bug reports. It makes it annoying to work with in most SGF editors -- its somewhat of a stress case.

Reply:
I'm not sure. 

I think we'd have to ask @anoek over in OGS why the SGF is generated like that. IIRC, OGS only publicly publishes frontend code and not backend code so I don't think I can investigate why OGS SGF generation works that way. 

I grabbed a game from my record on OGS and used it. I could pull something from KGS I suppose.

Reply:
@apetresc,  It turns out you aren’t the only person to ask this question. https://github.com/online-go/online-go.com/issues/2005 I just piled on and asked there as well.